versions/1/kibana/spec.yml

```diff
@@ -65,3 +65,25 @@
         type: file
         contentMediaType: "application/json"
         pattern: '^.+\.json$'
+    - description: Folder containing rules
+      type: folder
+      name: "rules"
+      required: false
+      contents:
+      - description: An individual rule file for the detection engine
+        type: file
+        contentMediaType: "application/json"
+        pattern: '^{PACKAGE_NAME}-rule-.+\.json$'
+      - description: Changelog for all of the rules in the folder
+        type: file
+        contentMediaType: "application/json"
+        pattern: '^{PACKAGE_NAME}-CHANGELOG\.json$'
+    - description: Folder containing timeline templates
+      type: folder
+      name: "timelines"
+      required: false
+      contents:
+      - description: A Kibana timeline template
```

Comment:
I'm not aware of differences between rules and timelines, but according to the description they're bound together? Is the `timeline` a Kibana object type?

Reply:
Some rules have dependencies on timelines, and right now in Kibana, there is an interrelationship between the two.

In Kibana today, we compile both into the release and from the Detection Engine page, users currently have one button to separate both. Maybe "detection_engine" is a better name for the package, since rules, timelines and ML jobs are all relevant and coupled.



@FrankHassanabad, @spong and @andrew-goldstein could add more detail.